src/main/java/Duke.java

```diff
@@ -1,10 +1,133 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
You could consider moving `Ui` and `Command` code to separate classes to better apply OOP principles.